src/NUnitFramework/framework/Constraints/RangeConstraint.cs

```diff
@@ -72,7 +87,7 @@ public override ConstraintResult ApplyTo<TActual>(TActual actual)
         {
             if ( from == null || to == null || actual == null)
                 throw new ArgumentException( "Cannot compare using a null reference", "actual" );
-
+            CompareFromAndTo();
```

Comment:
In my test, if I remove the check in ApplyTo it has the following failure
```
            Assert.Throws<ArgumentException>(() => test.Using(rComparer));
            Assert.Throws<ArgumentException>(() => test.Using(rComparer).ApplyTo(7));
            Assert.Throws<ArgumentException>(() => test.ApplyTo(7)); //fails now
```
But I suppose in a normal use case, the failed line would never have a chance to occur?